packages/wp-now/src/main.ts

```diff
@@ -1,6 +1,6 @@
 import { runCli } from './run-cli';
 
-const requiredMajorVersion = 18;
+const requiredMajorVersion = 20;
```

Comment:
In the future, can we handle version bumps as a separate PR?

Related https://github.com/WordPress/wordpress-playground/pull/591#pullrequestreview-1506199319